types/mongodb/index.d.ts

```diff
@@ -58,13 +59,136 @@ export class MongoClient extends EventEmitter {
     logout(options?: { dbName?: string }): Promise<any>;
     logout(options: { dbName?: string }, callback: MongoCallback<any>): void;
     /** http://mongodb.github.io/node-mongodb-native/3.1/api/MongoClient.html#startSession */
-    startSession(options?: any): ClientSession;
-}
+    startSession(options?: SessionOptions): ClientSession;
+    /** http://mongodb.github.io/node-mongodb-native/3.1/api/MongoClient.html#watch */
+    watch(pipeline?: Object[], options?: ChangeStreamOptions & { startAtClusterTime?: Timestamp, session?: ClientSession }): ChangeStream;
+    /** http://mongodb.github.io/node-mongodb-native/3.1/api/MongoClient.html#withSession */
+    withSession(operation: (session: ClientSession) => Promise<any>): Promise<void>;
+    /** http://mongodb.github.io/node-mongodb-native/3.1/api/MongoClient.html#withSession */
+    withSession(options: any, operation: (session: ClientSession) => Promise<any>): Promise<void>;
```

Comment:
FTR `options` is a `SessionOptions` object, but that is currently not properly documented

Reply:
thanks for the info, resolved in 787be30